src/main/kotlin/guru/stefma/androidartifacts/MavenPublication.kt

```diff
@@ -98,6 +96,34 @@ internal fun MavenPublication.setupMetadata(
     version = project.version as String
     artifactId = extension.artifactId
     groupId = project.group as String
+
+    if (GradleVersionComparator(project.gradle.gradleVersion).betterThan("4.7")) {
+        pom.name.set(extension.name ?: project.name)
+        pom.description.set(extension.description ?: project.description)
+        pom.url.set(extension.url)
+
+        // Add deprecated license property if available
```

Comment:
Do we really want to deprecate it?
I wouldn't do it for now.
Can you please remove the "deprecated" word here.

Reply:
```suggestion
        // Add license property if available
```